hbase_master/README.md

```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
 
 2. [Restart the Agent][9]
 
+### Log collection
+
+_Available for Agent versions >6.0_
+
+1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `hbase_master.d/conf.yaml` file to start collecting your Active Directory Logs:
+
+   ```yaml
+   logs:
+     - type: file
+       path: /path/to/my/directory/file.log
+       source: hbase
+       service: "<MY_SERVICE>"
```

Comment:
```suggestion
```

we don't need this here as we want customers to start using the service option in the init_config